løsningsforslag/R1/R1_18V/R1_18V_lf.tex

```diff
@@ -0,0 +1,408 @@
+% ---------------------------------------------------------------------
+% HEADER
+% Formålet med å legge header til et eget dokument er å garantere at
+% oppsettet av dokumentene er likt for alle løsningsforslagene.
+% I headeren skjer følgende:
+% (1) Dokumentet blir startet
+% (2) Pakker blir importert
+% ---------------------------------------------------------------------
+\input{../../header.tex}
+
+
+% ---------------------------------------------------------------------
+% DOKUMENTVARIABLER
+% ---------------------------------------------------------------------
+\newcommand{\fagkode}{R1}
+\newcommand{\semesteraar}{våren 2018}
+\newcommand{\forfatter}{Anita G.}
+\newcommand{\dokumenttittel}{Løsningsforslag -- Eksamen \fagkode, \semesteraar}
+
+\usepackage{siunitx}
```

Comment:
Hva gjør denne? Hvis den er nyttig kan du legge den inn med de andre pakkene i `header.tex`.

Reply:
Altså hvis den kan være nyttig i andre løsningsforslag.

Reply:
Det er en pakke for enheter. Jeg bruker den i dette løsningsforslaget for å få gradtegnet til en vinkel. Så den brukes her kun til en spesifikk ting.